shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -648,6 +676,14 @@ public void onMessage(Object message, BasicMessageChannel.Reply<Object> reply) {
         }
     }
 
+    private void createWindowChangeEvent(SemanticsObject route) {
+        // TYPE_WINDOW_STATE_CHANGED events should always be sent from the root node.
```

Comment:
Looks like this comment is out of date? You are using the specific node id now.

Reply:
Fixed